client/components/web-preview/index.jsx

```diff
@@ -110,6 +114,16 @@ export class WebPreview extends PureComponent {
 			</RootChild>
 		);
 	}
+
+	render() {
+		const { isContentOnly } = this.props;
+
+		if ( isContentOnly ) {
+			return this.renderContentOnly();
+		}
+
+		return this.renderPreviewWithContent();
+	}
```

Comment:
This is a good attempt to export both `WebPreview` and `WebPreviewContent` functionality as one async-loaded component, but needs some more work. The lifetime methods of `WebPreview` add an ESC key listener to the window and `no-scroll is-previewing` CSS classes to the document. That should happen only when `isContentOnly` is `false`, but is happening always.